helical/models/geneformer/fine_tuning_model.py

```diff
@@ -22,8 +22,8 @@ class GeneformerFineTuningModel(HelicalBaseFineTuningModel):
     ----------
     geneformer_model : Geneformer
         The initialised Geneformer model to fine-tune.
-    fine_tuning_head : HelicalBaseFineTuningHead
-        The fine-tuning head that is appended to the model.
+    fine_tuning_head : Literal["classification"] | HelicalBaseFineTuningHead
+        The fine-tuning head that is appended to the model. This can either be a string specifying the task or a custom fine-tuning head inheriting from HelicalBaseFineTuningHead.
```

Comment:
Specify which strings are possible. I guess only classification so far? Let's say that then